pcdsdevices/epics_motor.py

```diff
@@ -1470,7 +1473,7 @@ def done_comparator(self, readback, setpoint):
         return setpoint-self.atol < readback < setpoint+self.atol
 
 
-class SmarAct(EpicsMotorInterface):
+class SmarAct(PCDSMotorBase):
```

Comment:
Does the SmarAct class have the same oddities that the PCDSMotorBase was created to fix?
```
        1. The 'TDIR' field does not exist on the PCDS implementation. This
        indicates what direction the motor has travelled last. To account for
        this difference, we use the :meth:`._pos_changed` callback to keep
        track of which direction we believe the motor to be travelling
        and store this in a simple :class:`~ophyd.signal.Signal`.
        2. The 'SPG' field implements the three states used in the LCLS
        motor record.  This is a reduced version of the standard EPICS
        'SPMG' field.  Setting to 'STOP', 'PAUSE' and 'GO'  will respectively
        stop motor movement, pause a move in progress, or resume a paused move.
        3. In some cases, puts to the setpoint PV cannot be waited on. In fact
        this seems to brick the whole python session, requiring a slightly
        modified set_current_position method.
```
If not, we should instead be moving these all the way up to `EpicsMotorInterface`.

Reply:
Ah, you're right. It does not. I'll move them.